frontend/server/src/main/java/org/pytorch/serve/wlm/WorkerLifeCycle.java

```diff
@@ -39,6 +40,45 @@ public Process getProcess() {
         return process;
     }
 
+    public ArrayList<String> launcherArgsToList() {
+        ArrayList<String> arrlist = new ArrayList<String>();
+        arrlist.add("-m");
+        arrlist.add("intel_extension_for_pytorch.cpu.launch");
+        arrlist.add("--ninstance");
+        arrlist.add("1");
```

Comment:
is "--ninstance" always 1? If not, please make it configurable and update doc to specify the relationship b/w torchserve worker and ipex ninstance.